Ubuntu.Dockerfile

```diff
@@ -30,8 +30,10 @@ COPY . .
 
 
 #Install npm
-RUN npm ci --only=production
 RUN npm install
+RUN npm ci --only=production
+RUN npm install --save form-data
```

Comment:
Is it necessary to have? If yes, let's add it to package.json

Reply:
Added to package.json